lib/internal/errors.js

```diff
@@ -47,7 +47,6 @@ class AssertionError extends Error {
       throw new exports.TypeError('ERR_INVALID_ARG_TYPE', 'options', 'object');
```

Comment:
IMHO in the signature replace `constructor(options)` with `constructor({message, actual, operator, expected, stackStartFunction})`